client/package.json

```diff
@@ -78,6 +78,7 @@
     "rooks": "5.10.0",
     "sharp": "^0.31.1",
     "tailwindcss": "3.2.4",
+    "use-debounce": "^9.0.3",
```

Comment:
if I'm not wrong rooks package already includes useDebounce, could you check it?